Assignment more crud

benjaminmodayil opened the pull request.

alexstophel commented on .gitignore:
```diff
@@ -14,3 +14,6 @@
 /tmp
+
+# Ignore Figaro's application.yml file
+/config/application.yml
```
We should add this so your API keys aren't available online.

alexstophel commented:
I removed application.yml and squashed my commit into yours to remove the file from the history on Github. We don't want anyone else to see your API keys for security reasons. I'm going to go ahead and merge. Be sure to ```git pull origin master``` on the ```master``` branch before starting the next checkpoint

benjaminmodayil commented:
Okay, I believe I got it all working. Something went wrong initially when I
was going in between two computers, but it all seems to be working again.
Thanks.

On Sat, Jun 13, 2015 at 12:27 PM, Alex Stophel <notifications@github.com>
wrote:

> I removed application.yml and squashed my commit into yours to remove the
> file from the history on Github. We don't want anyone else to see your API
> keys for security reasons. I'm going to go ahead and merge. Be sure to git
> pull origin master on the master branch before starting the next
> checkpoint
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/benjaminmodayil/bloccit/pull/1#issuecomment-111727224>
> .
>

alexstophel commented:
Cool!
